dashboard/state_aggregator.py

```diff
@@ -195,6 +197,35 @@ async def list_objects(self, *, option: ListApiOptions) -> dict:
         result.sort(key=lambda entry: entry["object_id"])
         return {d["object_id"]: d for d in islice(result, option.limit)}
 
+    async def list_runtime_envs(self, *, option: ListApiOptions) -> List[dict]:
+        """List all runtime env information from the cluster.
+
+        Returns:
+            A list of runtime env information in the cluster.
+            We don't have id -> data mapping like other API because runtime env
```

Comment:
the serialized_runtime_env acts like the key (id)?

Reply:
yeah, should/could we generate a unique_id?

Reply:
Yeah serialized_runtime_env is used as an unique id within the implementation. Do you guys think it is a good id? I feel a bit weird to use this as an id (it could be sometimes really long too). Adding a new unique id would require a bit of refactoring.

Reply:
Actually, the serialized_runtime_Env is not unique because they are not unique across nodes. It's possible two nodes have the same serialized runtime env. 

@SongGuyang do we have a concept of id in the runtime env now?

Reply:
We generate a hash in the Raylet now. You can see https://github.com/ray-project/ray/blob/master/src/ray/common/task/task_spec.cc#L177. But it contains scheduling resources which is not a part of runtime_env. We can't use it directly.

Reply:
Hmm should we generate a random id and pass around?